### PR TITLE
[webgl]Remove cpu backend.

### PR DESCRIPTION
**Pull request by lina128**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4920)

### CODE REVIEW

**Comment by lina128** [comment]
cc @qjia7

File: tfjs-backend-webgl/src/backend_webgl.ts [hunk 1]
```diff
@@ -167,9 +166,7 @@ export class MathBackendWebGL extends KernelBackend {
   }
 
   numDataIds() {
-    return this.texData.numDataIds() +
-        (this.cpuBackend ? this.cpuBackend.numDataIds() : 0) -
```
Comment (qjia7) [inline tfjs-backend-webgl/src/backend_webgl.ts:171]: Hi Na, I am wondering if it is possible that `cpuBackend.numDataIds()` is not zero if some operators are using cpu. In this case, will `numDataIds()` be correct for webgl? If it doesn't exist, your change looks good for me. Thanks.

**Review by lina128 — COMMENTED** [review]
Hi Jiajia, good question. For webgl backend, after modularization, the operators that use cpu will no longer forward the data to cpu backend, instead, it simply uses the vanilla Javascript implementation imported from cpu package, e.g.: https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgl/src/kernels/Abs.ts#L36

Because data never leaves webgl backend, we don't need to count cpu backend anymore. 

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4920#-:-MY35wHX7lwue3DrjGW5:b9tx1l5)* status: 0 of 1 approvals obtained (waiting on @pyu10055)

**Comment by qjia7** [comment]
> Hi Jiajia, good question. For webgl backend, after modularization, the operators that use cpu will no longer forward the data to cpu backend, instead, it simply uses the vanilla Javascript implementation imported from cpu package, e.g.: https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgl/src/kernels/Abs.ts#L36
> 
> Because data never leaves webgl backend, we don't need to count cpu backend anymore.
> 

Make sense for me. Thanks for the explaination.

**Review by pyu10055 — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4920)* status: :shipit: complete! 1 of 1 approvals obtained

**Pull request merged**